descqa/ColorDistribution.py

```diff
@@ -217,8 +226,8 @@ def make_plots(self, mock_color_dist, color_shift, cvm_omega, cvm_omega_shift, c
             title = '$m_r < {:2.1f},  {:.1f} < z < {:.1f}$'.format(self.obs_r_mag_limit, self.zlo, self.zhi)
         elif self.Mag_r_limit:
             title = '$M_r < {:2.1f},  {:.1f} < z < {:.1f}$'.format(self.Mag_r_limit, self.zlo, self.zhi)
-
-        print(mock_color_dist.keys())
+        title = title + ' (LSST filters)' if filters == 'lsst' else title
```

Comment:
why not just always print out filters? 
```suggestion
        title += ' ({})'.format(filters.upper())
```

Reply:
This would be misleading in the case that the filters have undergone a color transformation. I added some code to include information about a filter transformation, if it occurred. Here's the example [DESCQA test](https://portal.nersc.gov/project/lsst/descqa/v2/?run=2019-08-09_9) with the revised code.